crates/rolldown/src/module_loader/module_loader.rs

```diff
@@ -178,11 +179,20 @@ impl ModuleLoader {
               },
             }
           };
+
+          self.symbol_ref_db.store_local_db(
+            idx,
+            SymbolRefDbForModule::new(SymbolTable::default(), idx, ScopeId::new(0)),
+          );
+          let symbol_ref = self.symbol_ref_db.create_facade_root_symbol_ref(
+            idx,
+            legitimize_identifier_name(resolved_id.id.as_str()).into(),
+          );
           let ext = ExternalModule::new(
             idx,
             ArcStr::clone(&resolved_id.id),
             external_module_side_effects,
-            SymbolNameRefToken::new(idx, legitimize_identifier_name(&resolved_id.id).into()),
+            symbol_ref,
```

Comment:
Here pervious using `SymbolNameRefToken`, it is different with `SymbolRef`, because here need to linker it to `NamespaceAlias#SymbolRef`, so here reback using `SymbolRef`.

here we could to change `NamespaceAlias#SymbolRef` to  enum `SymbolNameRefToken/SymbolRef` to be avoid here using `SymbolRef`. It is some complex so it is not including it at here.